### PR TITLE
Sp3-66:Added feature to change the account status 

### PR DESCRIPTION
**Pull request by AyushGiri190**
{DF8B2C96-5B15-475F-841B-B033AD53C303}
{DA0B0635-B536-445D-9CE3-0B8476597FAA}

### CODE REVIEW

File: src/components/business/table/BusinessTable.tsx [hunk 1]
```diff
@@ -67,6 +72,55 @@ const BusinessTable: React.FC = () => {
...
+    if (selectedIndex === -1) {
+      toast({
+        title: "Error",
+        description: "No user selected for status update.",
+        variant: "destructive",
+      });
+      return;
+    }
+  
+    const status = userData[selectedIndex].status;
```
Comment (Divesh-singh) [inline src/components/business/table/BusinessTable.tsx:91]: Unused variable

Comment (AyushGiri190) [reply src/components/business/table/BusinessTable.tsx:91]: It is being use in the catch part if there is an api error in updating then we need to retract the status to the previous status

File: src/components/business/table/BusinessTable.tsx [hunk 2]
```diff
@@ -203,6 +270,16 @@ const BusinessTable: React.FC = () => {
           </div>
         </Card>
       </div>
+      <ConfirmationDialog
+        isOpen={dialogOpen}
+        onConfirm={handleStatusUpdate}
+        onCancel={() => setDialogOpen(false)}
+        title="Confirm Updation"
+        description={`Are you sure you want to change the account status to "${selectedStatus}"`}
+        confirmButtonName="Confirm"
```
[the remark this answers is not in the document]
Comment (AyushGiri190) [reply src/components/business/table/BusinessTable.tsx:279]: Sorry about that forgot that

File: src/components/business/table/BusinessTable.tsx (hunk 1 above)
Comment (Divesh-singh) [reply src/components/business/table/BusinessTable.tsx:91]: okay

**Pull request merged**